src/main/java/com/blackducksoftware/integration/alert/web/channel/actions/ChannelDistributionConfigActions.java

```diff
@@ -251,18 +249,27 @@ public String validateConfig(final CommonDistributionConfig restModel, final Cha
 
     @Override
     public String testConfig(final CommonDistributionConfig restModel, final ChannelDescriptor descriptor) throws IntegrationException {
-        return distributionChannelManager.sendTestMessage(restModel, descriptor);
+        // TODO second expression is a current workaround to allow Slack to have a global config to display in UI and still let tests work properly.
+        if (descriptor.getConfig(DescriptorConfigType.GLOBAL_CONFIG) != null && (descriptor.getConfig(DescriptorConfigType.GLOBAL_CONFIG).getRepositoryAccessor() != null)) {
```

Comment:
We should change the About Controller to return the list of channels and add about to the UI.  Leave this TODO here but that would solve that issue.  It's probably a day or two at the most to add a new about screen.

Reply:
I've added an issue to Jira to track this